cloud/azure/app-services/detectors-app_services.tf

```diff
@@ -0,0 +1,174 @@
+resource "signalfx_detector" "heartbeat" {
```

Comment:
Add `threshold` function & use `lasting` function instead of aggregation function for App Service detectors.